configure.ac

```diff
@@ -272,6 +272,7 @@ UNICODE_LIBS=$UNISTRING_LIBS
 AC_SUBST(UNICODE_LIBS)
 
 WITH_LIBNL
+AM_CONDITIONAL([HAVE_LIBNL], test x"$HAVE_LIBNL" = x1)
```

Comment:
For the sake of consistency, I'd put this into conf_macros.m4::AC_DEFUN([WITH_LIBNL] (as with other 'WITH_')

Reply:
That is not possible because of the way `AM_CHECK_LIBNL3` and `AM_CHECK_LIBNL1` and declared and evaluated. They are different to the rest.
It seems the `AM_CONDITIONAL` statement needs to be evaluated after `WITH_LIBNL`. That's why I put it right after.
Declaring it in `libnl.m4` guarantees it will be evaluated before `WITH_LIBNL` and will always evaluate to `false`.

Reply:
I meant at the end of `AC_DEFUN([WITH_LIBNL]` in 'conf_macros.m4', not in 'libnl.m4'

Reply:
Ah! Sorry. I misunderstood you comment.

I think this should work.

Reply:
Done